### PR TITLE
Split controls pseudoauton

### PR DESCRIPTION
**Pull request by scarmain**
Merging changes we have been doing to mainline.  I am not planning to delete the branch yet, as we are both still developing on it.

### CODE REVIEW

**Review by programmer94 — COMMENTED** [review]
Are we sure about merging this right now? Note the spots where we haven't added any embedded autonomous instructions in tele-op.

**Comment by scarmain** [comment]
> 
> 
> Are we sure about merging this right now? Note the spots where we haven't added any embedded autonomous instructions in tele-op.

Yes, I am sure we can merge right now.  We took a different strategy to make the shooter autonomous work.  We are still using those stubs you created to stop the subsystems, so they are fine to leave.

We created a new class instead (ShooterAuton.java), and put all the logic for shooting in there.  The only thing it currently does is hold start, and the robot aims.  The drivers are currently using it.  Things we need to add (I'm a little scared on this work right now, we have 4 practices till Kettering, and I'm missing one of them):
* Variable speed aiming.  (Right now it's really slow, so could we say the further from center we are, go faster?)
* Check if wheels are up to speed
* Auto shoot
* Tune shoot wheels to see if we will make the target.

I guess the fear is keep master up to date, or risk losing changes when someone checkouts master.  We have been yelled at for not maintaining master, so I am trying to keep it up to date with functioning code.  (Ms. Carlini and I validated this code worked Thurs.)

Steven

**Review by programmer94 — APPROVED** [review]
This has my approval.

**Pull request merged**
